coldfront/core/project/management/commands/pending_join_request_reminder.py

```diff
@@ -0,0 +1,64 @@
+from coldfront.core.utils.common import import_from_settings
+from coldfront.core.utils.mail import send_email_template
+from coldfront.core.project.models import ProjectUserJoinRequest
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.urls import reverse
+from urllib.parse import urljoin
+import logging
+
+from coldfront.core.utils.common import utc_now_offset_aware
+from datetime import datetime
+from datetime import timedelta
+
+"""An admin command that sends PIs reminder emails of pending join requests."""
+
+TIME_DELTA = timedelta(days=4)
+
+class Command(BaseCommand):
+
+    help = (
+        'Send PIs reminder emails of pending join requests.')
+    logger = logging.getLogger(__name__)
+
+    def handle(self, *args, **options):
+
+        proj_join_requests_qeuryset = ProjectUserJoinRequest.objects.filter(project_user__status__name='Pending - Add')
```

Comment:
Thats true, but now we are just using this query set to find all projects and users with pending join requests. Having a previously denied request and a new pending request should not affect this.

Reply:
I also changed altered some query sets below in this [commit](https://github.com/ucb-rit/coldfront/pull/302/commits/8f0c2044090e2a89df302824978514432fdf79d0) so that only the latest request is shown in an email if there were previous denied requests.